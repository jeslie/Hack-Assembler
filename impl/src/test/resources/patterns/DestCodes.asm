// mnemonic // 111 0 000000 dst 000
    D&A     //              000
    M=D&A   //              001
    D=D&A   //              010
    MD=D&A  //              011
    A=D&A   //              100
    AM=D&A  //              101
    AD=D&A  //              110
    AMD=D&A //              111
