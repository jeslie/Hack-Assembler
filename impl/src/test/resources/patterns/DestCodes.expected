1110000000000000
1110000000001000
1110000000010000
1110000000011000
1110000000100000
1110000000101000
1110000000110000
1110000000111000
